Sources/PayPal/Buttons/PayPalButton.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+/// Configuration for PayPal button
+public class PayPalButton: PaymentButton {
+
+    // MARK: - Init
+
+    public init() {
+        super.init(frame: .zero)
+
+        translatesAutoresizingMaskIntoConstraints = false
+        backgroundColor = getButtonColor(for: .payPal)
+        setImage(getButtonLogo(for: .payPal), for: .normal)
+    }
+
+    @available(*, unavailable)
```

Comment:
Do we need this available check? I'm not sure what this is doing.

Reply:
I am not sure if the available check is necessary, but assume it is for something since Xcode defaults this implementation to our class if we remove this init from the class.